var/spack/repos/builtin/packages/libfabric/package.py

```diff
@@ -191,6 +196,9 @@ def configure_args(self):
         else:
             args.append("--with-kdreg=no")
 
+        if "+uring" in self.spec:
+            args.append("--with-uring=yes")
```

Comment:
We are now recommending use of `self.spec.satisfies()` to check variants in general but this one MAY be unique enough for using `in`.  The docs (https://spack.readthedocs.io/en/latest/packaging_guide.html#testing-spec-constraints) state:

"The difference between the two functions is that `satisfies()` tests whether spec constraints overlap at all, while `in` tests whether a spec or any of its dependencies satisfy the provided spec."